File: LeetCode Solutions/Problems/1798. Maximum Number of Consecutive Values You Can Make..java
class Solution {
    public int getMaximumConsecutive(int[] coins) {
        Arrays.sort(coins);
        int n = coins.length;
        int current = 1;
        for (int i=0; i<n; i++){
            if (coins[i] <= current){
               current += coins[i];
            } else{
                break;
            }
        }
        return current;
    }
}

File: LeetCode Solutions/Problems/1798. Maximum Number of Consecutive Values You Can Make.py
class Solution:
    def getMaximumConsecutive(self, coins: List[int]) -> int:
        coins = sorted(coins)
        currentMax = 0
        for c in coins:
            if currentMax + 1 < c:
                break
            currentMax = currentMax + c
        return currentMax + 1